### ISSUE TITLE
Elite Aspect equipment

### ISSUE THREAD

**Issue by gaforb** [WondaMegapon/Refightilization#1]
When spawning, give the monster players the elite aspect equipment instead of manually adding the buff. Allows for compatibility with AspectAbilities and so on, and might fix the issue of overlays not appearing for the player?
Unsure if this would overwrite their existing equipment slot

**Comment by WondaMegapon** [comment]
Ooo! Good idea, though overwriting existing equipment is an issue. What if the player has an empty equipment slot, they're eligible for being an elite?

**Comment by gaforb** [comment]
Then people would usually not be an elite... Could try to give them the equipment if they have the slot free, and the just the affix buff otherwise?
Best thing would be if you could somehow store the old equipment and give it back on the next stage. This is definitely a dumb workaround, but reading the equipment ID and giving the player that many stacks of a generic buff, then giving them back their old equipment and clearing the buff might work? Could make the buff icon a little crate or something.

**Comment by WondaMegapon** [comment]
Sorry for taking months on this one, but I've settled on a the compromise of adding the config option `ForceGrantAffix` to the mod. When enabled, affixes will override the player's current equipment, and return their previous equipment on a successful respawn.

I kept running into the thought that it'd be bad to forcibly take the player's items away, but then I realized I could just push the responsibility of deciding that behavior onto the host. 😛
